assets/js/components/post-searcher.js

```diff
@@ -112,28 +115,37 @@ class PostSearcher extends Component {
 	}
 
 	onConfirm( selection ) {
-		this.setState( {
-			selection,
-		} );
+		const { results } = this.state;
+		let match;
```

Comment:
This outer declaration is unnecessary as it's only set once. This can simply be `const match = results.find` below.

Reply:
Updated.